src/components/MyPersona/MyPersona.tsx

```diff
@@ -145,7 +145,7 @@ export const MyPersona: FC = () => {
           </TextInShare>
         </ShareQr>
       </PageWrapperSpaceBetween>
-      <NavLink to="./createpersona">
+      <NavLink to="./creation/step/1/entity/persona">
```

Comment:
please use APP_ROUTES constant from routes.ts here, or create one and use id here if there is no link already prepared